dcos_launch/config.py

```diff
@@ -196,16 +197,20 @@ def get_validated_config(user_config: dict, config_dir: str) -> dict:
     'key_helper': {
         'type': 'boolean',
         'default': False},
-    'zen_helper': {
-        'type': 'boolean',
-        'default': False},
     'tags': {
         'type': 'dict',
-        'required': False}}
+        'required': False},
+    'zen_helper': {
```

Comment:
zen is only applicable to AWS CF zen templates, so why does it get placed in "COMMON"?

Reply:
I just moved it back where it used to be but that doesn't mean it makes any more sense https://github.com/dcos/dcos-launch/blob/master/dcos_launch/config.py#L199. Just moved it now to another spot that makes sense.